client/api/python/heketi/heketi.py

```diff
@@ -23,6 +23,12 @@
 import json
 
 
+# Constants related to setting tags
+TAGS_SET = 'set'
```

Comment:
If you could also comment why these three lines are required here?

Reply:
I've been annoyed that the pyhton api code does not provide constants. This is the start of that. Instead of "remembering" the right strings for everything a set of constants remind one what options are available and misspelling of a constant will be an early error rather than a later confusion about what didn't work. I hope that explanation makes sense.

Reply:
Thanks, that explains :)